ibis/backends/spark/tests/test_udf.py

```diff
@@ -7,7 +7,7 @@
 import ibis.common.exceptions as com
 import ibis.expr.datatypes as dt
 import ibis.expr.types as ir
-from ibis.tests.backends import Spark
+from ibis.backends.spark.tests.conftest import SparkTest
```

Comment:
hmm isn't this a pytest warning? we generally don't want to do this

Reply:
e.g. just put the methods in another file (not conftest)

Reply:
I'm ok changing this, but pytest is happy, no warnings. And in several cases there is already related code in `conftest.py`. For example, in bigquery the `connect` methods of the tests class is already in conftest. And in omnisci and others, the parameters for the connection are loaded in conftest and reused for both the test class and for things in conftest. That's why I thought that conftest is a better place (even if in couple of instances like this one conftest needs to be imported). And also because the class is not tests themselves but more the configuration (even if not pytest specific).

So, to me it makes a bit more sense to leave the test classes in `conftest.py`, but if you have a strong opinion on having it separate, I'm ok with that, I'll move it next week. Let me know if you have a preference for the file where to save it (since I need to create 12 of them, to not waste time with renames later).

Reply:
nope it's fine

in pandas we prohibit this but maybe that's some enforced outside of pytest i don't remember